web/server/README.md

```diff
@@ -57,6 +57,30 @@ The API requests are serviced as follows:
 - `badges` gives access only to the badges API calls.
 - `management` gives access only to the management API calls.
 
+### Enabling TLS support
+
+Netdata since version 1.16 has the feature to encrypt connections. To enable this option it is necessary to have a certificate and a key, case you do not have one yet, it is possible to generate them  with the following command
+
+$ openssl req -newkey rsa:2048 -nodes -sha512 -x509 -days 365 -keyout key.pem -out cert.pem
+
+, it is important to say that the previous command will create a self signed certificate and the browsers will show warnings about it.
+
+To start using HTTPS with Netdata it is only necessary to write the path for your certificate (`ssl certificate`) and key(`ssl key`) inside the `web section`
+
```

Comment:
need to say somewhere here that the netdata user should be able to read both files (explain chown and chmod commands needed)

Reply:
I will redo the README when we're done